samples/test/create_realm.test.js

```diff
@@ -29,6 +30,11 @@ describe('Game Servers Create Realm Test', () => {
   const client = new RealmsServiceClient();
   let realmId;
 
+  before(async () => {
+    const projectId = await client.getProjectId();
+    await cleanup(client, null, projectId, LOCATION);
```

Comment:
should stop passing parameters now that it doesn't accept any :) Here and below.

Reply:
Done